buildSrc/src/main/kotlin/com/datadog/gradle/plugin/srpayloadupdate/SrPayloadCreateTask.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.gradle.plugin.srpayloadupdate
+
+import org.gradle.api.tasks.Exec
+
+open class SrPayloadCreateTask : Exec() {
+
+    init {
+        group = "datadog"
+        description = "Update the Session Replay integration tests payloads"
+        setCommandLine(
+            "adb", "shell", "am", "instrument", "-w", "-r",
+            "-e", "debug", "false",
+            "-e", "updateSrPayloads", "true",
+            "-e", "package", "com.datadog.android.sdk.integration.sessionreplay",
+            "com.datadog.android.sdk.integration.test/androidx.test.runner.AndroidJUnitRunner"
+        )
+    }
+
+    override fun exec() {
+        println("Executing command: ${commandLine.joinToString(" ")}")
```

Comment:
should use Gradle `logger` here instead of `println`, this will allow to control the verbosity.

Reply:
oh yeah...good point